tests/Integration/Rules/NoNullReturnRuleIntegrationTest.php

```
@@ -0,0 +1,82 @@
+        copy(
+            __DIR__ . '/../../Fixtures/Rules/NoNullReturnRule/SuppressedNull.php',
+            $copy
+        );
+
+        exec(
+            sprintf(
+                '%s analyse --error-format raw %s',
+                escapeshellcmd($this->phpstanBin),
+                escapeshellarg($copy)
+            ),
+            $output,
```

PHPMD (2.15.0)

Avoid unused local variables such as '$output'. (undefined)

(UnusedLocalVariable)